Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/AddOns/SceneExporter/constants.py

```diff
@@ -20,13 +20,18 @@
 NO_ANIMATION = 'No Animation'
 KEY_FRAME_ANIMATION = 'Keyframe Animation'
 MESH_AND_RIG = 'Just Mesh with Rig'
+SKIN_ATTACHMENT = 'Skin Attachment Mesh with Rig'
 ANIMATION_LIST_OPTIONS = ( (
     ('0', NO_ANIMATION, 'Export with no keyframe Animation.'),
     ('1', KEY_FRAME_ANIMATION, 'Mesh needs to be parented to Armature with weights in order for O3DE to detect Entity as an Actor.'),
-    ('2', MESH_AND_RIG, 'Key All Bones, Force exporting at least one key of animation for all bones')
+    ('2', MESH_AND_RIG, 'Key All Bones, Force exporting at least one key of animation for all bones'),
+    ('3', SKIN_ATTACHMENT, 'Export a mesh with the Armature bones for use as a O3DE Skin Attachment.')

```

Comment:
consider ending final lines in the collection with comma. python allows for trailing comma to reduce churn in future edits avoiding forgetting to add commas and having multiple lines change in diff.

Reply:
Thanks! That's a good power tip. Done.